File: ui/src/main/java/com/midtrans/sdk/corekit/models/FreeText.java
package com.midtrans.sdk.corekit.models;

import androidx.annotation.NonNull;
import java.util.List;

public class FreeText extends com.midtrans.sdk.uikit.api.model.FreeText {
    public FreeText(@NonNull List<? extends FreeTextLanguage> inquiry, @NonNull List<? extends FreeTextLanguage> payment) {
        super(inquiry, payment);
    }
}

File: ui/src/main/kotlin/com/midtrans/sdk/uikit/api/model/FreeText.kt
package com.midtrans.sdk.uikit.api.model

import com.midtrans.sdk.corekit.api.model.FreeText

open class FreeText(
    inquiry: List<FreeTextLanguage>,
    payment: List<FreeTextLanguage>
) : FreeText(
    inquiry = inquiry,
    payment = payment
)